apps/wallpaper/js/share.js

```diff
@@ -26,9 +26,11 @@ window.onload = function() {
   var scale;
   var previewImage = document.getElementById('previewImage');
 
-  // How big (in device pixels) is the screen?
-  var screenWidth = Math.ceil(screen.width * window.devicePixelRatio);
-  var screenHeight = Math.ceil(screen.height * window.devicePixelRatio);
+   // How big (in device pixels) is the screen?
+  var screenWidth = Math.ceil(Math.min(screen.width, screen.height) *
+                    window.devicePixelRatio);
+  var screenHeight = Math.ceil(Math.max(screen.width, screen.height) *
+                     window.devicePixelRatio);
```

Comment:
Nit: please fix the indentation on lines 31 and 33 so that 'window.device' lines up with Math.min and Math.max